### DISCUSSION TITLE
사용자 식별 값으로 아이디, 닉네임, 이름, 이메일 중 어떤 명칭을 사용할지 논의합니다.

### DISCUSSION THREAD

**Discussion by zeus6768** [woowacourse-teams/2024-code-zap#632] [결정 사항]
DB의 primary key에 해당하는 ID와는 무관합니다. 

### 로그인 화면
<img width="475" alt="Screenshot 2024-09-11 at 1 31 06 PM" src="https://github.com/user-attachments/assets/16174e94-339f-44cd-b9cc-1bfbf21db628">

### 필드명
![Screenshot 2024-09-11 at 1 31 41 PM](https://github.com/user-attachments/assets/8439dcd2-dbf0-4e9a-a4bd-ba14b88478eb)

## 안건
- 프론트와 백엔드에서 사용할 명칭을 결정합니다.

**Comment by Hain-tain** [comment]
언어 통일에 대한 합의 너무 좋습니다!!

### 프론트에서 `아이디(닉네임)` 으로 표기하게 된 배경은 아래와 같습니다. (개인적인 견해 95%)
- 우리 서비스에서 로그인을 위한 식별자로 아이디를 사용하기로 하였다. 
  - 아이디라고 명명한 이유는 현재 한 번 생성된 아이디는 변경이 불가능 하기 때문이다. 닉네임이라고만 표기한다면 추후 변경이 가능할 것 같은 느낌이 든다고 생각했다.
- 그런데 아이디라고만 해두니 한글 입력이 가능하다는 것을 알기 어렵다는 생각이 들었다.
  - 아이디를 만들라고 하면 보통 자신이 사용하던 영어+숫자 조합의 아이디를 떠올리기 때문이다.
- 가입을 한 뒤 한글 아이디를 발견하게 되면 자신도 바꾸고 싶지만 현재 아이디를 바꿀 수 없기 때문에 아이디가 닉네임처럼 사용된다는 것을 알려주기 위해 `아이디(닉네임)` 으로 표기해보았다.

### 사용자 식별 값으로 뭘 사용할까?
- 사용자 식별 값으로 `아이디` 에 한 표를 줍니다. 닉네임은 부가 설명일 뿐 본질은 아이디에 더 가깝다고 생각하기 때문입니다!
